book.rb

```diff
@@ -0,0 +1,16 @@
+class Book
+  attr_accessor :title, :author
+
+  @books = []
```

Comment:
- [ ] I'm not able to create a book because there's typo it should be `@@books = [ ]` instead `@books = [ ]`. Please fix this issue, this is how we initialized class variables. Creating a book is not working because you are calling `@@books` on line 9 and it does not exist.